rca/programmes/models.py

```diff
@@ -518,7 +518,27 @@ class ProgrammePage(BasePage):
         ]
     )
 
-    search_fields = BasePage.search_fields + []
+    search_fields = BasePage.search_fields + [
+        index.RelatedFields(
+            "programme_type",
+            [
+                index.RelatedFields(
+                    "programme_type",
+                    [
+                        index.SearchField("display_name", partial_match=True),
+                        index.AutocompleteField("display_name", partial_match=True),
+                    ],
+                )
+            ],
+        ),
+        index.RelatedFields(
+            "degree_level",
+            [
+                index.SearchField("title", partial_match=True),
+                index.AutocompleteField("title", partial_match=True),
+            ],
+        ),
+    ]
```

Comment:
We probably also want the other fields displayed in the results to be added here, so the `programme_description_subtitle` and (if possible?) the pathway blocks’ label?.

Reply:
Which one is the pathways block label?

![image](https://user-images.githubusercontent.com/2128707/71718763-2a2c6e80-2e14-11ea-9a11-c7bf17c004b6.png)

Assuming it's `Extend and develop...`?

Reply:
I've indexed the whole block as it's a streamfield, not an FK rel, ie all one blob in the db.